flank-scripts/src/main/kotlin/flank/scripts/integration/CommitList.kt

```diff
@@ -0,0 +1,26 @@
+package flank.scripts.integration
+
+import com.github.kittinunf.result.getOrElse
+import com.github.kittinunf.result.onError
+import flank.scripts.github.getGitHubCommitList
+import flank.scripts.github.getPrDetailsByCommit
+import kotlinx.coroutines.async
+import kotlinx.coroutines.awaitAll
+import kotlinx.coroutines.coroutineScope
+
+suspend fun getCommitListSinceDate(token: String, since: String) = coroutineScope {
```

Comment:
```suggestion
suspend fun getCommitListSinceDate(
    token: String, 
    since: String
): List<Pair<String, GithubPullRequest?>> = coroutineScope {
```